app/managers.py

```diff
@@ -0,0 +1,38 @@
+from app.models import Actor
+import sqlite3
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect("cinema.db3")
+
+    table_name = "actors"
+
+    def all(self) -> list[Actor]:
+        cursor = self._connection.execute(f"SELECT * FROM {self.table_name}")
+        return [Actor(*row) for row in cursor]
+
+    def create(self, first_name_: str, last_name_: str) -> None:
```

Comment:
```suggestion
    def create(self, first_name: str, last_name: str) -> None:
```
No need to add this _ at the end

Reply:
No,it`s need:     def create(self, first_name_: str, last_name_: str) -> None:
        self._connection.execute(
            f"INSERT INTO {self.table_name} "
           -> f"(first_name, last_name) VALUES (?, ?)",
            (first_name_, last_name_)

Reply:
Why you need _ at the end?
It is redundant it will work without it